lib/agent/actions.js

```diff
@@ -93,50 +106,53 @@ var start = function(type, name, opts, cb){
       // trigger this event after 10 seconds.
 
       setTimeout(function() {
-        hooks.trigger(type, 'stopped', name, opts);
+        hooks.trigger(type, 'stopped', id, name, opts);
       }, 10000);
 
     }
   })
 
 };
 
-actions.start = function(name, opts) {
-  start('action', name, opts);
+actions.start = function(id, name, opts, cb) {
+  start('action', id, name, opts, cb);
 }
 
-actions.start_trigger = function(name, opts, cb) {
-  start('trigger', name, opts, cb);
-}
+// agregar lo del opts para el response
```

Comment:
Si se van a dejar comentarios así, mejor usar TO DO y en inglés idealmente.
Igual creo que es más fácil borrarlo.
```suggestion
// TODO: agregar lo del opts para el response
```